### PR TITLE
Adds persistence and sending of feedback

### PR DESCRIPTION
**Pull request by eirism**
Lecture sessions and counts of each feedback type are saved in the DB.
Adds socketio handlers to process incoming events from students and lectures.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -6,3 +6,10 @@
 DEBUG = os.environ.get('DEBUG', 0)
 SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
+    ['action_slow', 'To slow'],
```
Comment (sardred) [inline config.py]: Too*

File: config.py [hunk 2]
```diff
@@ -6,3 +6,10 @@
 DEBUG = os.environ.get('DEBUG', 0)
 SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
```
Comment (plusk) [inline config.py:10]: This should be more consistent. Either say "Too slow, too fast, too easy, too hard", or be more verbose and say "this is too _something_".

Comment (plusk) [reply config.py:10]: This is a discussion we should take as a group too, considering I'd personally prefer just saying "fast, slow, easy, hard", but considering this is a "bot", we might want to be more communicative with the user.

**Review by plusk — DISMISSED** [review]
Other than this, it looks alright.

**Review by plusk — APPROVED** [review]

**Pull request merged**